docs/getting-started/install-configure.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Install & Configure
+toc: true
+weight: 1
+indent: true
+---
+
+# Install & Configure Quickstart
+
+Crossplane can be easily installed into any existing Kubernetes cluster using
+the regularly published Helm chart. The Helm chart contains all the custom
+resources and controllers needed to deploy and configure Crossplane.
+
+See [Install] and [Configure] docs for installing alternate versions and more
+detailed instructions.
+
+## Get a Kubernetes Cluster
+
+<ul class="nav nav-tabs">
+<li class="active"><a href="#setup-mac-brew" data-toggle="tab">macOS via Homebrew</a></li>
+<li><a href="#setup-mac-linux" data-toggle="tab">macOS / Linux</a></li>
+<li><a href="#setup-mac-linux" data-toggle="tab">Windows</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="setup-mac-brew" markdown="1">
+```
+brew upgrade
+brew install kind
+brew install kubectl
+brew install helm
+
+kind create cluster --image kindest/node:v1.15.11 --wait 5m
```

Comment:
We may want to recommend at least 1.17 here; the OAM local addon docs state they require it (and I'm inclined to believe them).

Reply:
done